test/new_relic/newrelic_rpm_test.rb

```diff
@@ -5,81 +5,21 @@
 require_relative '../test_helper'
 
 class NewRelicRpmTest < Minitest::Test
+  RAILS_32_SKIP_MESSAGE = 'MySQL error for Rails 3.2 when we add an initializer to config/initializers'
   # This test examines the behavior of an initializer when the agent is started in a Rails context
   # The initializer used for these tests is defined in test/environments/*/config/inititalizers/test.rb
 
   # We have documentation recommending customers call add_method_tracer
   # in their initializers, let's make sure this works
-  def test_add_method_tracer_in_initializer_gets_traced_when_agent_initialized_before_config_initializers
-    skip unless defined?(Rails::VERSION)
-    skip 'MySQL error for Rails 3.2' if Rails::VERSION::MAJOR == 3
-
-    with_config(defer_rails_initialization: false) do
-      assert Bloodhound.newrelic_method_exists?('sniff'),
-        'Bloodhound#sniff not found by' \
-        'NewRelic::Agent::MethodTracer::ClassMethods::AddMethodTracer.newrelic_method_exists?'
-      assert Bloodhound.method_traced?('sniff'),
-        'Bloodhound#sniff not found by' \
-        'NewRelic::Agent::MethodTracer::ClassMethods::AddMethodTracer.method_traced?'
-    end
-  end
-
   def test_add_method_tracer_in_initializer_gets_traced_when_agent_initialized_after_config_initializers
     skip unless defined?(Rails::VERSION)
-    skip 'MySQL error for Rails 3.2 if we define an initializer in config/initializers' if Rails::VERSION::MAJOR == 3
-
-    with_config(defer_rails_initialization: true) do
-      assert Bloodhound.newrelic_method_exists?('sniff'),
-        'Bloodhound#sniff not found by' \
-        'NewRelic::Agent::MethodTracer::ClassMethods::AddMethodTracer.newrelic_method_exists?'
-      assert Bloodhound.method_traced?('sniff'),
-        'Bloodhound#sniff not found by' \
-        'NewRelic::Agent::MethodTracer::ClassMethods::AddMethodTracer.method_traced?'
-    end
-  end
-
-  # All supported Rails versions have the default value for
-  # timestamped_migrations as true. Our initializer sets it to false.
-  # Test to resolve: https://github.com/newrelic/newrelic-ruby-agent/issues/662
-
-  def test_active_record_initializer_config_change_saved_when_agent_initialized_after_config_initializers
-    skip unless defined?(Rails::VERSION)
-    # TODO: This test passes in a Rails console in a playground app and in the customer's environment
-    # but fails in this unit test context
-    skip if Gem::Version.new(NewRelic::VERSION::STRING) >= Gem::Version.new('8.13.0')
```

Comment:
This test was removed because
1) it has never worked and was always being skipped
2) it is not possible for this test to work because the rails instrumentation is loaded prior to any tests running in the env tests, therefore configuration options that are used while loading instrumentation cannot be changed and tested in this way.